fieldpath/set_test.go

```diff
@@ -755,3 +756,255 @@ func TestSetNodeMapIterate(t *testing.T) {
+				MakePathOrDie("spec", "containers", 1, "resources", "limits"),
+				MakePathOrDie("spec", "containers", 1, "resources", "limits", "cpu"),
+			),
+		},
+		{
+			name: "container resize fields: filter status and metadata",
+			input: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("status"),
+				MakePathOrDie("metadata"),
+			),
+			filter: NewIncludeMatcherFilter(MakePrefixMatcherOrDie("spec", "containers", MatchAnyPathElement(), "resources")),
+			expect: NewSet(
+				MakePathOrDie("spec"),
+			),
+		},
+		{
+			name: "container resize fields: filter non-container spec fields",
+			input: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("spec", "volumes"),
+				MakePathOrDie("spec", "hostNetwork"),
+			),
+			filter: NewIncludeMatcherFilter(MakePrefixMatcherOrDie("spec", "containers", MatchAnyPathElement(), "resources")),
+			expect: NewSet(
+				MakePathOrDie("spec"),
+			),
+		},
+		{
+			name: "container resize fields: filter non-resource container fields",
+			input: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("spec", "containers"),
+				MakePathOrDie("spec", "containers", 0, "image"),
+				MakePathOrDie("spec", "containers", 0, "workingDir"),
+				MakePathOrDie("spec", "containers", 0, "resources"),
+			),
+			filter: NewIncludeMatcherFilter(MakePrefixMatcherOrDie("spec", "containers", MatchAnyPathElement(), "resources")),
+			expect: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("spec", "containers"),
+				MakePathOrDie("spec", "containers", 0, "resources"),
+			),
+		},
+		{
+			name: "filter listMap key",
+			input: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("spec", "listMap",
+					&value.FieldList{
+						{Name: "key1", Value: value.NewValueInterface("value1")},
+						{Name: "key2", Value: value.NewValueInterface("value2")},
+					}),
+				MakePathOrDie("spec", "listMap",
+					&value.FieldList{
+						{Name: "key1", Value: value.NewValueInterface("value1")},
+						{Name: "key2", Value: value.NewValueInterface("value2")},
+					}, "field"),
+				MakePathOrDie("spec", "listMap",
+					&value.FieldList{
+						{Name: "key1", Value: value.NewValueInterface("valueX")},
+						{Name: "key2", Value: value.NewValueInterface("valueY")},
+					}, "field"),
+			),
+			filter: NewIncludeMatcherFilter(MakePrefixMatcherOrDie("spec", "listMap", &value.FieldList{
+				{Name: "key1", Value: value.NewValueInterface("value1")},
+				{Name: "key2", Value: value.NewValueInterface("value2")},
+			})),
+			expect: NewSet(
+				MakePathOrDie("spec"),
+				MakePathOrDie("spec", "listMap",
+					&value.FieldList{
+						{Name: "key1", Value: value.NewValueInterface("value1")},
+						{Name: "key2", Value: value.NewValueInterface("value2")},
+					}),
+				MakePathOrDie("spec", "listMap",
+					&value.FieldList{
+						{Name: "key1", Value: value.NewValueInterface("value1")},
+						{Name: "key2", Value: value.NewValueInterface("value2")},
+					}, "field"),
+			),
+		},
+		{
+			name: "filter listMap key",
```

Comment:
I can't tell what is different between this case and the previous, with the same name?

Reply:
Thanks for catching.

Looks like I fat fingered something while doing my copy-paste a previous test then modify it to create my next test.  I've removed.